.github/workflows/pre-main.yaml

```diff
@@ -13,26 +13,17 @@ permissions:
 env:
   REGISTRY: quay.io
   REGISTRY_LOCAL: localhost
-  TNF_IMAGE_NAME: testnetworkfunction/cnf-certification-test
-  TNF_IMAGE_TAG: unstable
+  CERTSUITE_IMAGE_NAME: testnetworkfunction/cnf-certification-test
+  CERTSUITE_IMAGE_TAG: unstable
   OCT_IMAGE_NAME: testnetworkfunction/oct
   OCT_IMAGE_TAG: latest
-  TNF_CONTAINER_CLIENT: docker
-  TNF_NON_INTRUSIVE_ONLY: false
-  TNF_ALLOW_PREFLIGHT_INSECURE: false
-  TNF_DISABLE_CONFIG_AUTODISCOVER: false
-  TNF_CONFIG_DIR: /tmp/tnf/config
-  TNF_OUTPUT_DIR: /tmp/tnf/output
-  TNF_CERTSUITE_CMD_OUTPUT_DIR: /tmp/tnf/certsuite-out
-  TNF_SRC_URL: 'https://github.com/${{ github.repository }}'
-  TESTING_CMD_PARAMS: '-n host -i ${REGISTRY_LOCAL}/${TNF_IMAGE_NAME}:${TNF_IMAGE_TAG} -t ${TNF_CONFIG_DIR} -o ${TNF_OUTPUT_DIR}'
-  TNF_SMOKE_TESTS_LOG_LEVEL: debug
-  ON_DEMAND_DEBUG_PODS: false
+  CERTSUITE_CONFIG_DIR: /tmp/certsuite/config
+  CERTSUITE_OUTPUT_DIR: /tmp/certsuite/output
+  SMOKE_TESTS_LOG_LEVEL: debug
+  SMOKE_TESTS_LABELS_FILTER: all
   TERM: xterm-color
   CM_BIN: /usr/local/bin/checkmake
   CM_URL_LINUX: https://github.com/mrtazz/checkmake/releases/download/0.2.2/checkmake-0.2.2.linux.amd64 # yamllint disable-line
-  SMOKE_TESTS_LABELS_FILTER: all
-  SKIP_PRELOAD_IMAGES: true
```

Comment:
I think we need to keep this flag `SKIP_PRELOAD_IMAGES`.

Reply:
Done.